messaging/src/main/java/edu/tamu/weaver/messaging/annotation/WeaverMessageListener.java

```diff
@@ -0,0 +1,25 @@
+package edu.tamu.weaver.messaging.annotation;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import org.springframework.core.annotation.AliasFor;
+import org.springframework.jms.annotation.JmsListener;
+
+@Documented
+@JmsListener(destination = "")
+@Retention(RUNTIME)
+@Target({METHOD, TYPE})
```

Comment:
Does this annotation do anything when applied to a class? If not TYPE target should be removed.

Reply:
https://github.com/TAMULib/Weaver-Webservice-Core/commit/0136047545c9b6ee1ef128301a3aef53fb55b765